### PR TITLE
Eng-219/implement coingecko api key

### PR DESCRIPTION
**Pull request by StevanPataki99**

### CODE REVIEW

File: packages/indexer/.env.example [hunk 1]
```diff
@@ -120,6 +120,7 @@ BULLMQ_ADMIN_PASSWORD=
 METADATA_INDEXING_METHOD=
 METADATA_INDEXING_METHOD_COLLECTION=
 DISABLE_REALTIME_METADATA_REFRESH=0
+COINGECKO_API_KEY=
```
Comment (RobbyUitbeijerse) [inline packages/indexer/.env.example:123]: I think you need to do something with devops to make this available to the runtime process in AWS

File: packages/indexer/package.json [hunk 2]
```diff
@@ -72,6 +72,8 @@
     "arweave": "^1.10.23",
     "aws-kms-ethers-signer": "^0.1.3",
     "aws-sdk": "^2.1148.0",
+    "axios": "^1",
+    "axios-cache-interceptor": "^1",
```
Comment (RobbyUitbeijerse) [inline packages/indexer/package.json:76]: can you lock a more exact version?

^1.5.2

File: packages/indexer/package.json [hunk 3]
```diff
@@ -72,6 +72,8 @@
     "arweave": "^1.10.23",
     "aws-kms-ethers-signer": "^0.1.3",
     "aws-sdk": "^2.1148.0",
+    "axios": "^1",
```
Comment (RobbyUitbeijerse) [inline packages/indexer/package.json:75]: can you lock a more exact version?

^1.6.8

File: packages/indexer/src/utils/coingecko/index.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import { setupCache } from "axios-cache-interceptor";
+
+import { logger } from "@/common/logger";
+import { config } from "@/config/index";
+
+export type CGMarketData = {
+  market_data: {
+    current_price: { [symbol: string]: number };
```
Comment (RobbyUitbeijerse) [inline packages/indexer/src/utils/coingecko/index.ts:9]: it won't change anything, but you could use:
```
current_price: Record<string, number>
```

**Review by RobbyUitbeijerse — APPROVED** [review]

**Review by lewinskimaciej — APPROVED** [review]

**Pull request merged**